Sources/AudioKit/Nodes/Mixing/Mixer.swift

```diff
@@ -33,17 +37,18 @@ public class Mixer: Node, Toggleable {
     }
 
     /// Initialize the mixer node with no inputs, to be connected later
-    public init(volume: AUValue = 1.0) {
+    public init(volume: AUValue = 1.0, name: String? = nil) {
         super.init(avAudioNode: mixerAU)
         self.volume = volume
+        self.name = name ?? self.name
```

Comment:
I'm not sure what the goal was here, but this approach can be simplified to just `self.name = name` if you provide a default on the parameter and make it non-optional.